conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/ApiValidation.scala

```diff
@@ -0,0 +1,32 @@
+package tech.cryptonomic.conseil.api.routes.platform.data
+
+import akka.http.scaladsl.model.{ContentTypes, HttpEntity, StatusCodes}
+import akka.http.scaladsl.server.Directives.complete
+import akka.http.scaladsl.server.Route
+import akka.util.ByteString
+import endpoints.algebra.Documentation
+import tech.cryptonomic.conseil.api.routes.platform.data.CsvConversions._
+import tech.cryptonomic.conseil.common.generic.chain.DataTypes.{
+  OutputType,
+  QueryResponseWithOutput,
+  QueryValidationError
+}
+import tech.cryptonomic.conseil.common.util.Conversion.Syntax._
+
+object ApiValidation {
```

Comment:
Once more, maybe docs would greatly improve the reading experience and understanding